src/cap-description.adoc

```diff
@@ -81,6 +81,19 @@ The byte-address of a memory location is encoded as MXLEN integer value.
 ^| 64      ^| {cap_rv64_addr_width}
 |==============================================================================
 
+[#section_cap_level,reftext="capability level"]
+==== Capability Level (CL)
```

Comment:
What instruction could be used to read/write a capability's level?

Reply:
Added that it can be read with GCPERM.